### ISSUE TITLE
channels are reversed on visualize

### ISSUE THREAD

**Issue by rt-luxonis** [zylo117/Yet-Another-EfficientDet-Pytorch#214]
Hello
I am running the visualization part to maybe see why my coco_eval results are nearly 0 while my training and val loss is around 0.8 after 50 epochs.
This is my visualize output:
![image](https://user-images.githubusercontent.com/62758232/80397861-79b11180-88bf-11ea-8ae0-0b4b38e48e6f.png)

**Comment by zylo117** [comment]
1，0.8 val loss is quite high. it's very possible that you are suffering overfitting.
2. you should monitor tensorboard, the loss graph
